core/ledger/test/test.yaml

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+#
+#    Peer section
+#
+###############################################################################
+peer:
+    # Path on the file system where peer will store data
+    fileSystemPath: /var/hyperledger/test/ledger_test
```

Comment:
As a side note: I wish we had a way of being able to use relative paths in the YAMLs/tests. While your `fileSystemPath` is configured correctly, I checked after looking at this issue's changes, and I found that at the project levels not all the absolute paths are. Relative paths or concatenated ones, at least, would have made it better.